src/main/java/es/thalesalv/gptbot/adapters/discord/listener/SlashCommandListener.java

```diff
@@ -0,0 +1,55 @@
+package es.thalesalv.gptbot.adapters.discord.listener;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.BeanFactory;
+import org.springframework.beans.factory.NoSuchBeanDefinitionException;
+import org.springframework.stereotype.Service;
+
+import es.thalesalv.gptbot.application.service.commands.CommandService;
+import jakarta.annotation.Nonnull;
+import lombok.RequiredArgsConstructor;
+import net.dv8tion.jda.api.events.interaction.ModalInteractionEvent;
+import net.dv8tion.jda.api.events.interaction.command.SlashCommandInteractionEvent;
+import net.dv8tion.jda.api.hooks.ListenerAdapter;
+
+@Service
+@RequiredArgsConstructor
+public class SlashCommandListener extends ListenerAdapter {
+
+    private final BeanFactory beanFactory;
+
+    private static final String NON_EXISTING_COMMAND = "The command you tried to use does not exist. Please use `create`, `delete` or `update` as the argument.";
+    private static final String LOREBOOK_ENTRY_SERVICE = "LorebookEntryService";
+    private static final Logger LOGGER = LoggerFactory.getLogger(SlashCommandListener.class);
+
+    @Override
+    public void onSlashCommandInteraction(SlashCommandInteractionEvent event) {
+
+        try {
+            LOGGER.debug("Received slash command event -> {}", event);
+            event.deferReply();
+            final String eventName = event.getName();
+            if (eventName.equals("lorebook")) {
```

Comment:
I feel like some of these strings should be externalized for easier maintenance. Like if we wanted to change an eventName in the future? Might be a little pedantic, but I would do that.

Reply:
Wouldn't that bloat the configuration files? We can do that, but I really think they won't be changed that often to make sense to have them externalized.

Reply:
I just meant to maybe have a relevant Constants file somewhere so that we could easily change the name of something globally w/o search/replace. Maybe it would never be needed, so it's fine to leave as-is. It was just an observation more than a "you can't do this."